packages/feedback-service/src/e2e/mock.json

```diff
@@ -0,0 +1,4 @@
+{
+   "message": "Define your test mocks in this file and remove this key value pair if not required"
```

Comment:
This file should include the mock data for your tests. So add an object with the feedback schema and test values to be used in your unit tests.

How are your tests passing without the mock??